### PR TITLE
Implement Facade Blocks

### PR DESCRIPTION
**Pull request by woodiertexas**

### CODE REVIEW

File: src/main/resources/assets/architecture_extensions/templates/model/block/template_facade.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      "faces": {
+        "north": {"uv": [0, 0, 16, 16], "texture": "#texture"},
+        "east": {"uv": [15.99, 0, 16, 16], "texture": "#texture"},
+        "south": {"uv": [0, 0, 16, 16], "texture": "#texture"},
+        "west": {"uv": [0, 0, 0.01, 16], "texture": "#texture"},
+        "up": {"uv": [0, 0, 16, 0.01], "texture": "#texture"},
+        "down": {"uv": [0, 15.99, 16, 16], "texture": "#texture"}
+      }
+    }
+  ],
```
Comment (maximumpower55) [inline src/main/resources/assets/architecture_extensions/templates/model/block/template_facade.json:20]:
```suggestion
  ],
  "gui_light": "front",
```

File: src/main/resources/assets/architecture_extensions/templates/model/block/template_facade.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
+{
+  "credit": "Made with Blockbench",
+  "parent": "architecture_extensions:block/template_lattice",
```
Comment (maximumpower55) [inline src/main/resources/assets/architecture_extensions/templates/model/block/template_facade.json:3]: Why is this parent still here? it doesn't do anything

File: src/main/resources/assets/architecture_extensions/templates/model/block/template_facade.json (hunk 1 above)
Comment (maximumpower55) [reply src/main/resources/assets/architecture_extensions/templates/model/block/template_facade.json:20]: This single new line will make the lighting on the model much better in GUI

File: src/main/resources/assets/architecture_extensions/templates/model/block/template_facade.json (hunk 2 above)
Comment (maximumpower55) [inline src/main/resources/assets/architecture_extensions/templates/model/block/template_facade.json]:
```suggestion
```

**Review by maximumpower55 — APPROVED** [review]
Everything looks good expect these 2 nitpicks.

File: src/main/java/io/github/debuggyteam/architecture_extensions/blocks/FacadeBlock.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import net.minecraft.block.LichenSpreadBehavior;
+import net.minecraft.block.ShapeContext;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.shape.VoxelShape;
+import net.minecraft.util.shape.VoxelShapes;
+import net.minecraft.world.BlockView;
+
+/**
+ * @author Gaming32 & Portal Cubed devs
+ * @since ${version}
```
Comment (falkreon) [inline src/main/java/io/github/debuggyteam/architecture_extensions/blocks/FacadeBlock.java]: this annotation should always be frozen to one version, and not updated to the latest by gradle.

**Review by falkreon — CHANGES_REQUESTED** [review]
PR looks and works great, I just want to do credits really well.

File: src/main/java/io/github/debuggyteam/architecture_extensions/blocks/FacadeBlock.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+ **/
+public class FacadeBlock extends AbstractLichenBlock {
+	public FacadeBlock(Settings settings) {
+		super(settings);
+	}
+
+	@Override
+	public VoxelShape getCollisionShape(BlockState state, BlockView view, BlockPos pos, ShapeContext context) {
+		return VoxelShapes.empty();
+	}
```
Comment (Ampflower) [inline src/main/java/io/github/debuggyteam/architecture_extensions/blocks/FacadeBlock.java:24]: Just want to be clear; facade is indeed meaning no collision?

Comment (woodiertexas) [reply src/main/java/io/github/debuggyteam/architecture_extensions/blocks/FacadeBlock.java:24]: Yea, that makes the collision for the block non-existent.

File: src/main/java/io/github/debuggyteam/architecture_extensions/blocks/FacadeBlock.java (hunk 3 above)
Comment (Ampflower) [reply src/main/java/io/github/debuggyteam/architecture_extensions/blocks/FacadeBlock.java]: Yeah, that kinda got carried over from here; it was never meant to be anything more than something that can be easily Ctrl+Shift+R then replaced when it was time to actually release the code.

**Review by Ampflower — APPROVED** [review]
Looks good to me

**Review by falkreon — APPROVED** [review]

**Pull request merged**